publications-2025.bib

```diff
@@ -1024,6 +1024,20 @@ @Article{2025:prieto-saavedra.munch.ea:matrix-free
   preprint = {https://ssrn.com/abstract=4981567}
 }
 
+@Article{2025:prieto-saavedra.archambault.ea:implicit,
+  author  = {Prieto Saavedra, Laura and Archambault, Jo{\"e}l and Munch, Peter and Blais, Bruno},
+  title   = {An implicit large-eddy simulation study of the turbulent Taylor-Couette flow with an inner rotating cylinder},
+  journal = {Journal of Turbulence},
+  year    = 2025,
+  volume  = 0,
+  pages   = 1--15,
+  month   = july,
+  issn    = {1468-5248},
+  url     = {https://www.sciencedirect.com/science/article/pii/S0021999125004693},
+  doi     = {10.1080/14685248.2025.2524336}
+  publisher = {Taylor \& Francis},
```

Comment:
You're missing a comma at the end of the doi entry.

Reply:
Nice catch. Will fix asap